server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -122,10 +123,20 @@ object NettyConfig {
     pipeline.addLast(new HttpObjectAggregator(cfg.maxContentLength))
     pipeline.addLast(new ChunkedWriteHandler())
     pipeline.addLast(handler)
+    ()
+  }
+
+  def streamingPipeline(cfg: NettyConfig)(pipeline: ChannelPipeline, handler: ChannelHandler): Unit = {
+    cfg.sslContext.foreach(s => pipeline.addLast(s.newHandler(pipeline.channel().alloc())))
+    pipeline.addLast(new HttpServerCodec())
+    pipeline.addLast(new HttpStreamsServerHandler())
+    pipeline.addLast(handler)
     if (cfg.addLoggingHandler) pipeline.addLast(new LoggingHandler())
     ()
   }
 
+  def defaultWithStreaming: NettyConfig = default.copy(initPipeline = cfg => streamingPipeline(cfg)(_, _))
```

Comment:
maybe we should be more explicit here and rename `default` to `defaultNoStreaming`, same with `defaultInitPipeliene`, so that there's no ambiguity